pcl_utilities/launch/voxel_grid_filter.xml

```diff
@@ -0,0 +1,12 @@
+<launch>
+    <node
+        pkg="pcl_utilities"
+        exec="voxel_grid_filter_service"
+        name="voxel_grid_filter"
+        namespace="robot_common_3d/pcl_utilities"
+    >
+        <param name="filters.voxel_grid.leaf_size_x" value="0.05" />
```

Comment:
I did not have the filters prefix but in ROS1 they weren't necessary, this seems to be part of the port to ROS2 as params now generally are assigned a node name prefix. There is a way around this if the params are generic/used across multiple nodes:

node1:
  ros_parameters:
    param1: <val>
/**:
  ros_parameters:
    param2: <val_2>

Where /** is literal and just assigns the ros parameters across all nodes. You can still assign the parameter a nodespace by prepending the nodespace like so:

/nodespace:
  node3:
    ros_parameters:
      param3: <val3>

If this is modified further, it might make sense to change your "namespace" argument to something like node name since there may be a namespace that the node is wrapped into, possibly creating confusion within the code.